packages/collaborative-editing-backend/.env.example

```
@@ -0,0 +1,6 @@
+HOST=0.0.0.0
+PORT=1234
+MONGODB_URL=mongodb://admin:admin@127.0.0.1:27017/?authSource=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MONGODB_URL key should go before the PORT key

(UnorderedKey)